Java/src/main/java/net/hypixel/api/reply/AbstractReply.java

```diff
@@ -18,12 +21,35 @@ public String getCause() {
         return cause;
     }
 
+    public int getRequestAmountLeft()
+    {
+        return limitLeft;
+    }
+
+    public void setRequestAmountLeft(int limitLeft)
+    {
+        this.limitLeft = limitLeft;
+    }
+
+    public int getSecondsTillReset()
+    {
+        return limitReset;
+    }
+
+    public void setSecondsTillReset(int limitReset)
+    {
+        this.limitReset = limitReset;
+    }
```

Comment:
First of all, "Until", please. Secondly, as stated above, remove the setters

Reply:
I will rename them, for the setters, see above.